source/leuven/services/openondemand.rst

```diff
@@ -436,16 +442,20 @@ You can do this by using the ``lib`` argument for both the ``install.packages``
   You will also notice that you cannot use the same way of navigating after this.
   Another solution is to click the three dots on the right (...) and enter your path.
 - The 'Tools-Install packages' interface does not allow you to select any other path than the default in your ``$VSC_HOME``.
-  It is recommended to use the ``install.packages`` function instead.
+  It is recommended to use the ``install.packages()`` function instead.
 - RStudioServer will by default store the RStudio cache in ``$VSC_HOME/.local/share/rstudio``.
   This cache can get very large, and cause you to exceed the quota of your home directory.
-  To avoid this, you can redirect this cache to your data directory by setting ``$XDG_DATA_HOME``
-  variables in your ``~/.bashrc``.
+  To avoid this, you can redirect this cache to your data directory by setting the ``$XDG_DATA_HOME``
+  variable in your ``~/.bashrc``:
 
   .. code-block:: bash
 
     echo "export XDG_DATA_HOME=$VSC_DATA/.local/share" >> ~/.bashrc
 
+- Additionally, it is advised to change the default behaviour of RStudio to not restore .RData
+  into the workspace on start up and to never Save the workspace to .RData on exit.
+  You can do this via the RStudio interface:
```

Comment:
... via the RStudio interface by going to '...'. 
please add the terminating point to the sentence